packages/cysync-core/src/pages/AddAccount/AddAccount.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {
+  joystickArrowIcon,
+  DialogBox,
+  DialogBoxHeader,
+  DialogBoxBody,
+  checkIcon,
+  StyledLeanBoxContainer,
+  LeanBox,
+  halfLoaderGold,
+  VerifyCoinIcon,
+  Typography,
+  Image,
+  Container,
+} from '@cypherock/cysync-ui';
+
+export const AddAccount: React.FC = () => {
+  const dataArray = [
+    {
+      id: '1', // Add a unique identifier to each data object
+      leftImageSrc: joystickArrowIcon,
+      rightImageSrc: checkIcon,
+      text: 'Verify the coins on the X1 Vault',
+    },
+    {
+      id: '2',
+      leftImageSrc: joystickArrowIcon,
+      rightImageSrc: halfLoaderGold,
```

Comment:
Since it is a loader, it should be rotating. Use `Throbber` component for loader